Add release notes for Frontend v3.10.0 (#2019)
alphagov/govuk-frontend: CHANGELOG.md

## Unreleased

## 3.10.0 (Feature release)

### New features

#### Update users with notification banners

> You can now [use notification banners](https://design-system.service.gov.uk/components/notification-banner/) to inform users about events that occur elsewhere in the service. For example, when they need to complete tasks.

Author: ```suggestion
You can now [use notification banners](https://design-system.service.gov.uk/components/notification-banner/) to tell users about events that occur elsewhere in your service. For example, when they need to complete tasks.
```

This was added in [pull request #1935: Create notification banner component](https://github.com/alphagov/govuk-frontend/issues/1935).

#### Customise input mode in the date component

> You can now enable mobile users to change their keyboard selection when they complete fields in the [date component](https://design-system.service.gov.uk/components/date-input/).

Author: ```suggestion
You can now change which keyboard type displays for mobile and tablet users. This enables users to enter characters other than numbers when they use the date input. To change the keyboard type, set the `inputmode` when you use the Nunjucks macro options.```
